docs/site/migration/auth/example.md

```diff
@@ -0,0 +1,570 @@
+also need to create repositories for data persistency. You can read about their
+difference in document
+[migrating model definitions and built-in APIs](https://loopback.io/doc/en/lb4/migration-models-core.html)_
+
+_To keep the tutorial concise and focus on the core implementation, the detailed
+commands are list in the reference [model creation](#model-creation)._
+
+## Set up login endpoint
+
+LoopBack 3 has a default User model with a bunch of pre-defined APIs exposed. In
+this example, since we create the User model from scratch, we need to add the
+required endpoints in the User controller. In the demo we only need
+'User/login'.
+
+This application uses token based authentication. A user logs in by providing
+correct credentials (email and password) in the payload of 'User/login', then
+gets a token back with its identity information encoded and includes it in the
+header of next requests.
+
+To create the login endpoint, we first run `lb4 controller` to create a User
+controller (see [user controller creation](#user-controller-creation)). Then add
+a new controller function `login` decorated with the REST decorators that
+describe the request and response (see
+[the complete user controller file](https://github.com/strongloop/loopback-next/blob/example/acl-migration/examples/access-control-migration/src/controllers/user.controller.ts)).
+The core logic of `login` does 3 things:
+
+- call `userService.verifyCredentials()` to verify the credentials and find the
+  right user.
+- call `userService.convertToUserProfile()` to convert the user into a standard
+  principal shared across the authentication and authorization modules.
+- call `jwtService.generateToken()` to encode the principal that carries the
+  user's identity information into a JSON web token.
+
+To keep the controller file concise and to organize the token and user related
+utils, we will create the token service and user service in section
+[Set up authentication](#set-up-authentication). Now you can just inject them.
+
+## Set up project endpoints
+
+Next let's create the 5 project endpoints which will be used to demo the access
+of different roles. We start from creating a controller for Project (see
+[project controller creation](#project-controller-creation)). Then incrementally
+add the following endpoints:
+
+- `/project/listProjects`: show all the projects but hide the `balance` field
+- `/project/viewAll`: show all the projects with the full information
+- `/project/{id}/findById`: show a project's information
+- `/projects/{id}/donate`: donate to a project
+- `/projects/{id}/withdraw`: withdraw from a project
+
+The complete code can be found in
+[src/controller/project.controller.ts](https://github.com/strongloop/loopback-next/blob/example/acl-migration/examples/access-control-migration/src/controllers/project.controller.ts)
+
+## Set up authentication
+
+This demo uses token based authentication, and uses the jwt authentication
```

Comment:
perhaps I don't know enough of LB3 or the access-control example... Where in https://github.com/strongloop/loopback-example-access-control indicates it's using JWT authentication or is it by default that JWT authentication is used when authentication is enabled?

Reply:
created story in https://github.com/strongloop/loopback-next/issues/4753